app/webpacker/styles/layout.scss

```diff
@@ -71,6 +71,7 @@ section.container {
   flex-wrap: wrap;
   flex-direction: column;
   margin: 1em 0;
+  word-break: break-word;
```

Comment:
this looked like a good solution to the problem.... unfortunately the setting `word-break: break-word;` is deprecated so we shouldn't use it in a new implementation. 

Have a look at `overflow-wrap: break-word;` instead?